WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -55,6 +55,12 @@ extension UIImage {
         UIImage(named: "blaze-intro-illustration")!
     }
 
+    /// Blaze product placeholder
+    ///
+    static var blazeProductPlaceholder: UIImage {
+        UIImage(named: "blaze-product-placeholder")!
```

Comment:
Placeholder comes from this discussion mBhyoV7OPhFnOugxLy6SMS-fi-2579:66616#663051121